javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,11 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var nameArray = [];
+  for (var i = 0; i < students.length; i++) {
+    for (var key in students[i]) {
+      var object = students[i];
+      nameArray.push(object[key]);
+    }
```

Comment:
Instead of creating an additional loop, you should just retrieve the value of the `name` property.